dataprep/eda/missing/compute/nullivariate.py

```diff
@@ -18,38 +18,103 @@
 def _compute_missing_nullivariate(df: DataArray, bins: int) -> Generator[Any, Any, Intermediate]:
     """Calculate the data for visualizing the plot_missing(df).
     This contains the missing spectrum, missing bar chart and missing heatmap."""
+    # pylint: disable=too-many-locals
+
+    most_show = 5
```

Comment:
Could you write a comment about the meaning of the two parameters here? Other parts looks good to me.